### PR TITLE
 Update according to comment on AUR Web

### PR DESCRIPTION
**Pull request by S7evinK**

### CODE REVIEW

**Comment by chris-morgan** [comment]
OK, I finally got back to this (funny how time flies).

- The conduit-example.toml patch needs to be modified: there’s a new line `database_backend = "rocksdb"` immediately below the `database_path` line.

- Having thought more about it, I prefer /etc/matrix-conduit/conduit.toml to /etc/matrix-conduit.toml: that way if more /etc files get added later, no change is required. Not a big deal by any means.

- I’m in favour of renaming the package from matrix-conduit to conduit. The word “matrix” in the name is basically noise; if there were already a package named conduit I could understand it, but when there isn’t I don’t think it’s necessary.

- I have now successfully built and installed it, and it’s running. Most functionality doesn’t seem to be working, but I’ll file issues on Conduit about that because I believe the problems are with it rather than with this package.

**Comment by S7evinK** [comment]
Thanks for the feedback (again), and sorry for my delay. 
I've talked to Conduits maintainer and he'd like to keep the word "matrix" in the package name.

Just pushed a small update, if you'd like to verify that the package is OK now. :)

File: PKGBUILD [hunk 1]
```diff
@@ -1,25 +1,37 @@
...
+  'SKIP'
+  '085e69c4589597f0b58509f72f04ecc497aaddcc5de6138f4ae5009f0d0cf4c9'
+  '88ff8d11d9617bc7af8025452d89231ae84f80605da3f183acf90bfddf57feda'
+  'c9f63b25049e51688911d8ec816fe5676d3cc70d889c321891e6f27888973ce9'
+)
+
+prepare() {
+  cd "$_pkgname"
+  patch --forward --strip=1 --input="${srcdir}/0001-update-service-dynamicuser_paths.patch"
+  patch --forward --strip=1 --input="${srcdir}/0002-example-info.patch"
```
Comment (Xiretza) [inline PKGBUILD:33]:
```
  cargo fetch --locked --target "$CARCH-unknown-linux-gnu"
```

File: PKGBUILD [hunk 2]
```diff
@@ -28,18 +40,13 @@ pkgver() {
 
 build(){
   cd "$_pkgname"
-  env CARGO_INCREMENTAL=1 cargo build --release --locked
+  env CARGO_INCREMENTAL=0 cargo build --release --locked
```
Comment (Xiretza) [inline PKGBUILD]:
```suggestion
  export RUSTUP_TOOLCHAIN=stable
  export CARGO_TARGET_DIR=target
  cargo build --frozen --release
```

File: PKGBUILD [hunk 3]
```diff
@@ -28,18 +40,13 @@ pkgver() {
...
 }
```
Comment (Xiretza) [inline PKGBUILD:55]:
```
check() {
  cd "$_pkgname"

  export RUSTUP_TOOLCHAIN=stable
  cargo test --frozen
}
```

**Review by Xiretza — CHANGES_REQUESTED** [review]
See the [Rust package guidelines](https://wiki.archlinux.org/title/Rust_package_guidelines). `--all-features` had to be removed from the cargo invocations because the conduit repo contains broken code gated by some non-default features (`heed` and `sled` at time of writing).

**Pull request merged**